### PR TITLE
Fixed the suggested issues

### PR DESCRIPTION
**Pull request by tanveerisonline**
### Following are the changes applied to this project :

1. Added changes to HTML and CSS. 
2. Added changes to CSS to follow the Dry principle (Don't Repeat Yourself Principle).
3. Fixed issues related to responsiveness.
4. Fixed the broken MIT license link.
5. Followed HTML and CSS best practices.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,53 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (ahmetbozaci) [inline README.md]: - [ ] This link is still broken. To make a professional readme file please fix it.

File: styles.css [hunk 2]
```diff
@@ -4,11 +4,21 @@
...
-  /* overflow-x: hidden; */
   font-family: "poppins", Helvetica, Arial, Verdana, Tahoma, sans-serif;
 }
 
+.d-flex {
+  display: flex;
+}
+
+.f-size {
+  font-size: 18px;
```
Comment (ahmetbozaci) [inline styles.css:19]: - Well done apply the DRY CSS principles 💯

File: README.md [hunk 3]
```diff
@@ -1,50 +1,53 @@
...
+Clone this repository by using the command line :
+`git clone https://github.com/tanveerisonline/personal-portfolio-website.git`
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (ahmetbozaci) [inline README.md]: ### OPTIONAL
- [ ] I recommend you fill many sections as you can. But if you won't fill sections better to delete them. It is important to make a professional readme file.

File: index.html [hunk 4]
```diff
@@ -4,27 +4,26 @@
...
 <body>
   <main class="container">
   
     <header class="header">
-      <nav class="desktop-navbar">
-        <div class="desktop-nav">
-          <a href="#" target="_parent" class="logo">Tanveer Ahmad</a>          
+      <nav class="desktop-navbar d-flex">
+        <div class="desktop-nav d-flex">
+          <a href="#" target="_parent" class="logo f-size">Tanveer Ahmad</a>
```
Comment (ahmetbozaci) [inline index.html]: - [ ] Website is still not responsive below 768px width remaining the same which is not good for tablets. Kindly fix this

1. You can see the container class has a width. That's why the remaining 375px. But not only this class. Please find which classes have a width and make the website unresponsive, delete them. If the page is still not responsive kindly find which CSS makes happen that and fix it.

1. Background image should responsive too.

image
___
After closed width
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
@tanveerisonline  ✋🏼 
### This is a manual review
### Please don't open a new branch for changes, continue with this branch(my-portfolio)   ⚠️ ⚠️ 
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️ 


#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
___ 

### 💯  Highlights 💯
- You follow the DRY CSS principles.
___

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by medaminedev66 — APPROVED** [review]
Hi @tanveerisonline 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- The page is responsive
- Your design looks great and matches the Figma design

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**